VenafiPS/Public/Get-VenafiCertificate.ps1

```diff
@@ -39,6 +42,15 @@ function Get-VenafiCertificate {
         [Alias('Path')]
         [string] $CertificateId,
 
+        [Parameter()]
+        [switch] $IncludePreviousVersions,
```

Comment:
new params need to be added to .parameter and .example.  also, need to let folks know they are tpp only params since this function targets vaas as well.

Reply:
Added